tests/ui_tests/test_adcm_upgrade.py

```diff
@@ -0,0 +1,108 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import threading
+from typing import Tuple
+
+import pytest
+import allure
+
+from version_utils import rpm
+from selenium.common.exceptions import StaleElementReferenceException
+from adcm_client.objects import ADCMClient
+from adcm_pytest_plugin.docker_utils import ADCM
+from adcm_pytest_plugin.plugin import parametrized_by_adcm_version
+from adcm_pytest_plugin.utils import wait_until_step_succeeds
+
+from tests.ui_tests.app.app import ADCMTest
+from tests.ui_tests.app.pages import BasePage
+from tests.ui_tests.app.page.admin.page import AdminIntroPage
+from tests.ui_tests.app.page.login.page import LoginPage
+from tests.ui_tests.app.page.cluster_list.page import ClusterListPage
+from tests.ui_tests.app.page.bundle_list.page import BundleListPage
+from tests.ui_tests.app.page.profile.page import ProfilePage
+
+# pylint: disable=redefined-outer-name
+
+
+@allure.step("Check that version has been changed")
+def _check_that_version_changed(before: str, after: str) -> None:
+    if rpm.compare_versions(after, before) < 1:
+        raise AssertionError("ADCM version after upgrade is older or equal to the version before")
+
+
+def old_adcm_image():
+    return parametrized_by_adcm_version(adcm_min_version="2021.03.10")[0][-1]
+
+
+@pytest.fixture(scope="session")
+def upgrade_target(cmd_opts) -> Tuple[str, str]:
+    if not cmd_opts.adcm_image:
+        pytest.fail("CLI parameter adcm_image should be provided")
+    return tuple(cmd_opts.adcm_image.split(":", maxsplit=2))  # type: ignore
```

Comment:
These methods are copy-paste of the ones from `tests/functional/test_adcm_upgrade.py`

Need to move it to the upper context and probably refactor it